config/example.yml

```diff
@@ -30,6 +30,8 @@ crowdin:
         locale_map:
             fr_FR: fr
             fr_BE: fr_BE
+        # You can specify a valid locale pattern to check after before renaming the files
+        valid_locale_pattern: /^[a-z]{2}_[A-Z]{2}$/
```

Comment:
Warning, I think there is some strange locales with 3 or 4 locale parameter

gsw-FR
kok-IN
quz-EC
arn-CL

Reply:
Ah...
I'll have to set the right pattern then. Thanks for the hint!